### PR TITLE
Update minimum go version to 1.15

### PR DESCRIPTION
**Pull request by praveenkumar**
- https://golang.org/doc/modules/gomod-ref#go

### CODE REVIEW

**Comment by praveenkumar** [comment]
https://github.com/code-ready/crc/issues/2473

File: Containerfile.ubuntu20.04 [hunk 1]
```diff
@@ -1,15 +0,0 @@
-FROM ubuntu:20.04
-
```
Comment (cfergeau) [inline Containerfile.ubuntu20.04:2]: I'm guessing it was problematic because of golang 1.15?

**Review by cfergeau — APPROVED** [review]

File: Containerfile.ubuntu20.04 (hunk 1 above)
Comment (praveenkumar) [reply Containerfile.ubuntu20.04:2]: @cfergeau Yes, that's right.

Comment (cfergeau) [reply Containerfile.ubuntu20.04:2]: Would be useful to have this explicityll in the commit log rather than leaving this as a guess.

**Pull request merged**